sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/DistinctMapTest.java

```diff
@@ -6,6 +6,7 @@
 import com.azure.cosmos.implementation.Utils;
 import com.azure.cosmos.implementation.query.DistinctMap;
 import com.azure.cosmos.implementation.query.DistinctQueryType;
+import com.azure.cosmos.implementation.routing.UInt128;
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
```

Comment:
Add tests for 1 vs 1.0 and escaped string

Reply:
Added integration test for this already in DistinctQueryTests#queryDocumentsForDistinctIntValues